src/core/diff.ts

```diff
@@ -106,22 +106,22 @@ export class Diff {
 
     switch (diffResponse.status) {
       case 200:
-        const version: VersionResponse = diffResponse.data;
+        const diff: WithDiff = diffResponse.data;
 
-        this.d(`Received version:`);
-        this.d(version);
-        return version;
+        this.d(`Received diff:`);
+        this.d(diff);
+        return diff;
         break;
       case 202:
         this.d('Waiting 1 sec before next pool');
```

Comment:
This is not related but I think this should be `poll` instead of `pool`

Reply:
oh, yes you are right I'll change it thanks